app/plugin.go

```diff
@@ -233,6 +233,9 @@ func (a *App) SyncPlugins() *model.AppError {
 
 	var wg sync.WaitGroup
 	for _, plugin := range availablePlugins {
+		if plugin.Manifest == nil {
```

Comment:
Should we push this back into `pluginsEnvironment.Available()`, and filter out anything without a manifest? I'd argue "it's not a plugin" without a manifest, so it should probably be suppressed everywhere.

Reply:
Sure, makes sense 👍